### PR TITLE
chore(config): Restore `expire_metrics` back to original type

### PR DESCRIPTION
**Pull request by bruceg**
This preserves the behavior present in version 0.24.0 to avoid breaking any configurations, while still marking the setting as deprecated.

### CODE REVIEW

File: website/cue/reference/configuration.cue [hunk 1]
```diff
@@ -41,15 +41,32 @@ configuration: {
...
+				remove all metrics that have not been updated in the given time.
 				This value must be positive.
 				"""
-			warnings: ["Deprecated, please use `expire_metrics_secs` instead."]
 			required: false
-			type: float: {
-				default: null
-				examples: [60.0]
-				unit: "seconds"
+			warnings: ["Deprecated, please use `expire_metrics_secs` instead."]
```
Comment (jszwedko) [inline website/cue/reference/configuration.cue:48]: 👍

**Review by jszwedko — COMMENTED** [review]
Awesome, thanks! We chatted in Slack but I think it makes sense to temporarily implement `Configurable` for `Duration` until we delete this field.

**Review by tobz — APPROVED** [review]
Makes enough sense to me. 👍🏻

**Review by jszwedko — APPROVED** [review]
Great, thanks!

**Pull request merged**